src/pages/[l1_l2]/[topic]/[...slug].tsx

```diff
@@ -93,6 +94,10 @@ const TopicContentTypePage: NextPage<TopicContentTypePageProps> = ({
         /^[aeiou]/gi.test(contentType) ? 'an' : 'a'
     } ${contentType}`; // Regex to tell if it starts with a vowel.
 
+    const contentTypeTitle =
+        contentTypeSlug.charAt(1).toUpperCase() + contentTypeSlug.slice(2);
```

Comment:
Why not just use `contentType` here? I think this logic will result in `Code Examples` having a title of `Atlas Code-examples | MongoDB`.